src/components/Highlight/Highlight.stories.tsx

```diff
@@ -0,0 +1,36 @@
+import { Meta, StoryObj } from "@storybook/react";
+import Highlight, { highlightColorList } from ".";
+
+export const Default: StoryObj<typeof Highlight> = {
+  render: (args) => (
+    <span>
+      Should be <Highlight {...args}>highlighted</Highlight>
+    </span>
+  ),
+};
+
+export const AllColors: StoryObj<typeof Highlight> = {
+  render: (args) => (
+    <div>
+      {highlightColorList.map((color) => (
+        <div key={color}>
+          Should be{" "}
+          <Highlight {...args} color={color}>
+            highlighted
+          </Highlight>
+        </div>
+      ))}
+    </div>
+  ),
+};
+const meta: Meta<typeof Highlight> = {
+  component: Highlight,
+  argTypes: {
+    color: {
+      control: "select",
+      options: highlightColorList,
+    },
+  },
```

Comment:
I tried interacting with this, but I'm not sure it does anything

```suggestion

```

Reply:
It does 
<img width="1142" alt="image" src="https://user-images.githubusercontent.com/4605522/219492854-59ea20d9-a0bd-454a-956c-7b1660add1cd.png">